src/doc/cli_manual/functions.rst

```diff
@@ -248,16 +242,16 @@ AlterDatabaseCorrelation
   AlterDatabaseCorrelation(name, databases, method) -> integer
 
 
-name : type
-    The name argument must be a string object containing the name of the database correlation to be altered.
+name : string
+    The name of the database correlation to be altered.
 
-databases : type
+databases : AMBIGUOUS
```

Comment:
Maybe `AMBIGUOUS` here is a keyword to facilitate manual patch-up of these cases after the change-over? Otherwise, maybe in this case use `VARIOUS (see description)`

Reply:
Yes, for the time begin AMBIGUOUS and STARTING_VALUE are meant to indicate that we need to manually review the text. These are just meant as placeholders to allow us to more easily find where this is needed.

Reply:
Sounds good. Great idea.